### DISCUSSION TITLE
"SX126X_SYNC_WORD_PRIVATE" was not declared in this scope

### DISCUSSION THREAD

**Discussion by frogg1ven** [jgromes/RadioLib#503] [General]
Hi,
after updating library from 4.5.0 to 5.1.2, I'm receiving this error: `"SX126X_SYNC_WORD_PRIVATE" was not declared in this scope`.
The same thing when I am trying to use `ERROR_NONE` or `ERR_CRC_MISMATCH`. I am guessing that all defined variables are unreachable.
I don't know if this was intended, or maybe it's only my problem, but I found that I can't use it now. (I'm using your library with Visual Studio Code and ESP32-WROOM-32UE)

**Comment by jgromes** [comment]
In 5.x, the defines changed to `RADIOLIB_*`, so `SX126X_SYNC_WORD_PRIVATE` became `RADIOLIB_SX126X_SYNC_WORD_PRIVATE`.

**Reply by frogg1ven** [reply]
thanks for reply
